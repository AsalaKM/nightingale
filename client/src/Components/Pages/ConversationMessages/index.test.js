import React from "react";
import renderer from "react-test-renderer";
import Messages from "./index.jsx";

describe("Messages", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Messages />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
