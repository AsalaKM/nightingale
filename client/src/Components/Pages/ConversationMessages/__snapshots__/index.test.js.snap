// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messages renders correctly 1`] = `
Array [
  <div
    className="sc-bdVaJa ctDytw"
  >
    <i
      aria-hidden="true"
      class="fa fa-angle-left"
      onClick={[Function]}
    />
    <p
      className="sc-bwzfXH iKrtUL"
    >
      
    </p>
    <p
      className="sc-bwzfXH iKrtUL"
    />
  </div>,
  <div
    className="sc-htpNat fkkHGS"
  >
    
    <div
      className="sc-bxivhb jANfsG"
    >
      <div>
        <p
          className="sc-ifAKCX gYPNTB"
        >
          
        </p>
        <p
          className="sc-ifAKCX gYPNTB"
        >
          
        </p>
      </div>
    </div>
  </div>,
]
`;
